Cargo.toml

```diff
@@ -18,7 +18,7 @@ tempfile = "2.1.5"
 slog = "2.1.1"
 slog-stdlog = "3.0.2"
 libloading = "0.4.0"
-wayland-client = { version = "0.20.5", optional = true }
+wayland-client = { version = "0.21.1", features = ["egl"], optional = true }
```

Comment:
I don't understand, why did you need to add this feature here?

Reply:
It's no longer a default feature since 0.21. But in fact I suppose that there is a good reason not to use 0.21 so I should probably remove this change from my PR.

Reply:
Ah right, I forgot about that. Okay! :+1: 

Upgrading wayland-client is completely fine, but in this case, could you also upgrade winit to 0.18 ? I believe the upgrade should be trivial.

Reply:
Yes, it just compiles and anvil seems to work normally.